packaging/testing/smoke/packages/Dockerfile.ubuntu2204

```diff
@@ -0,0 +1,32 @@
+# For staging upgrade we use the 'official-install' as the base
+
+# hadolint ignore=DL3007
+FROM dokken/ubuntu-22.04:latest as official-install
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+# Use the one-line install
+RUN curl https://raw.githubusercontent.com/fluent/fluent-bit/master/install.sh | sh
+RUN systemctl enable fluent-bit || systemctl enable td-agent-bit
+
+COPY ./test.sh /test.sh
+RUN chmod a+x /test.sh
+
+FROM official-install as staging-upgrade-prep
+RUN rm -f /etc/apt/sources.list.d/fluent-bit.list
+
+# hadolint ignore=DL3007
+FROM dokken/ubuntu-22.04:latest as staging-install
+ARG STAGING_URL
+ARG STAGING_VERSION
+ENV STAGING_VERSION=${STAGING_VERSION}
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN wget -qO - $STAGING_URL/fluentbit.key | apt-key add -
+RUN echo "deb $STAGING_URL/ubuntu/focal focal main" >> /etc/apt/sources.list
+RUN apt-get update && apt-get install --no-install-recommends -y td-agent-bit \
```

Comment:
@patrick-stephens should i ignore `DL3008` as well

Reply:
Yeah, that's fine for test images.